libres/lib/enkf/obs_data.cpp

```diff
@@ -334,6 +346,18 @@ obs_data_type *obs_data_alloc(double global_std_scaling) {
     return obs_data;
 }
 
+obs_data_type *obs_data_alloc_copy(const obs_data_type *src) {
```

Comment:
This function does not seem to be in use. Delete it?

Reply:
It is in use in #2981 - although maybe not in the sorry state of this actual PR.